src/client/authorization-code.ts

```diff
@@ -81,11 +93,43 @@ export class OAuth2AuthorizationCodeClient {
       code: params.code,
       redirect_uri: this.redirectUri,
       client_id: this.client.settings.clientId,
-      code_verifier: params.codeVerifier,
+      code_verifier: this.codeVerifier,
     };
     return tokenResponseToOAuth2Token(this.client.request('tokenEndpoint', body));
 
   }
 
 
 }
+
+export function getCodeVerifier(): string {
+  const arr = new Uint8Array(32);
+  crypto.getRandomValues(arr);
```

Comment:
Apparently this is only in Node v17 and newer? Is this package only supposed to work with >=v17?

Reply:
It will be Node 18 and up in fact! 17 is now dead, only the even numbers are LTS